lib/poker_hands/card.rb

```diff
@@ -0,0 +1,10 @@
+class Card
+    def initialize(rank, suit)
+        @rank = rank
+        @suit = suit
+    end
+
+    def get_card
+        @rank + @suit
+    end
+end
```

Comment:
By convention, we usually use a 2 space indentation (as opposed to 4).

http://www.caliban.org/ruby/rubyguide.shtml#indentation

Github is also complaining about a lack of newline at the end of the file, I believe you
can fix that with your editor 👍